src/ioctls/vm.rs

```diff
@@ -2513,6 +2554,20 @@ mod tests {
         assert!(vm.check_extension(Cap::MpState));
     }
 
+    #[test]
+    fn test_check_extension_int() {
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        assert!(vm.check_extension_int(Cap::MpState) > 0);
+    }
```

Comment:
Since `test_check_extension_raw` exists, in my view `test_check_extension_int` is not necessary, it's just a wrapper on `check_extension_raw`. This test is just duplicating existing code :)